SM300/iocBoot/iocSM300-IOC-01/st-common.cmd

```diff
@@ -0,0 +1,69 @@
+epicsEnvSet("DEVICE", "L0")
+< $(IOCSTARTUP)/init.cmd
+
+< $(IOCSTARTUP)/dbload.cmd
+
+# specify additional directories in which to to search for included request files
+set_requestfile_path("${MOTOR}/motorApp/Db", "")
+## as we are common, we need to explicity define the 01 area for when we are ran by 02, 03 etc 
+set_requestfile_path("${TOP}/iocBoot/iocSM300-IOC-01", "")
+
+iocshCmdLoop("< st-ctrl.cmd", "CNUM=\$(I)", "I", 1, 24)
+iocshCmdLoop("< st-max-axis.cmd", "MN=\$(I)", "I", 1, 8)
+
+epicsEnvSet("AMOTOR", "SM300MOTOR")
+
+## For recsim:
+$(IFRECSIM) motorSimCreateController("$(AMOTOR)", $(NAXES=1)) 
+$(IFRECSIM) drvAsynSerialPortConfigure("$(AMOTOR)", "NUL", 0, 1)
+
+
+## For dev sim devices
+$(IFDEVSIM) drvAsynIPPortConfigure("$(DEVICE)", "localhost:$(EMULATOR_PORT=)")
```

Comment:
Please can this default to 57677 so that I can run the emulator on a standard port number without having to edit the cmd.